### ISSUE TITLE
Transfer

### ISSUE THREAD

**Issue by noizu** [alexgaribay/sendgrid_elixir#47]
Are you open to transferring this to my noizu-labs org. I rely on this thing on a few projects.

**Comment by alexgaribay** [comment]
Yes. I'm willing to transfer as I don't have the time or desire to maintain this. What's your hex.pm username?

**Comment by noizu** [comment]
glad to help, noizulabs
